### ISSUE TITLE
[PAGE ISSUE]: 'A tour of the Dart language'

### ISSUE THREAD

**Issue by jaean123** [dart-lang/site-www#3672]
### Page URL

https://dart.dev/guides/language/language-tour

### Page source

https://github.com/dart-lang/site-www/tree/master/src/_guides/language/language-tour.md

### Describe the problem

Scrollbar on Chrome on Mac is not visible when it is on a white portion of the page. This is the case for most pages on dart.dev.

Screenshot:
https://ibb.co/BCjwhbp

### Expected fix

Please change the color of the scrollbar or keep the defaults.

**Comment by parlough** [comment]
Thanks for the report @jaean123! It's definitely important to be able to see and interact with the scrollbar, so I'll work on a fix soon.

After some investigation, I believe removing or changing the background-color property on the `html` css declaration in [main.css](https://github.com/dart-lang/site-www/blob/master/src/_assets/css/main.scss#L34) would fix this. I guess Chrome bases the scrollbar color off of that. We may need to make some other changes to avoid affecting styles elsewhere, but I'll investigate further soon.

**Comment by jaean123** [comment]
You're welcome. Thank you for your support.
